pikaday.js

```diff
@@ -656,6 +680,19 @@
         self.el = document.createElement('div');
         self.el.className = 'pika-single' + (opts.isRTL ? ' is-rtl' : '') + (opts.theme ? ' ' + opts.theme : '');
 
+        self.speakEl = document.createElement('div');
+				self.speakEl.setAttribute('role', 'status');
+				self.speakEl.setAttribute('aria-live', 'assertive');
+				self.speakEl.setAttribute('aria-atomic', 'true');
+        self.speakEl.setAttribute(
+          'id',
+          'boke-no-boke'
+        );
+				self.speakEl.setAttribute(
+					'style',
+					'position: absolute; left: -9999px; opacity: 0;'
```

Comment:
Seems that it's still take some height in the dom. I checked on my local, and it creates additional whitespace below footer

https://user-images.githubusercontent.com/28719400/106725058-e7a31a00-6643-11eb-9049-88e7a0b47954.mov